### PR TITLE
Update authorize.py

### PR DESCRIPTION
**Pull request by yunfachi**
fix

### CODE REVIEW

**Comment by AlekseyKorshuk** [comment]
According to YooMoney Docs `client_secret` is optional. Please make the following changes to the code.

**Comment by yunfachi** [comment]
check

**Comment by AlekseyKorshuk** [comment]
this will not work
because optional argument is not in the end

**Comment by AlekseyKorshuk** [comment]
please check before pushing any pull request

**Comment by yunfachi** [comment]
в начале `client_secret: str=None,`
в конце `f"{('&client_secret=' + client_secret) if client_secret!=None else ''}&grant_type=authorization_code&redirect_uri={redirect_uri}"`
немного не понял, что именно тебе не понравилось

File: yoomoney/authorize/authorize.py [hunk 1]
```diff
@@ -12,12 +12,13 @@ class Authorize:
     def __init__(
             self,
             client_id: str,
+            client_secret: str=None,
```
Comment (AlekseyKorshuk) [inline yoomoney/authorize/authorize.py:15]: SyntaxError: non-default argument follows default argument

**Pull request closed without merging**